From dcb3422530bcefd86653818230b0e6941440c70b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 30 Jan 2022 17:29:49 +0200
Subject: [PATCH] [SW-0] gaudi2: security: no redefine spi_flash_size

SPI_FLASH_SIZE is already defined in gaudi2.h included from specs.
No need to redefine it, so remove the SPI_FLASH_SIZE define.

Change-Id: Ie4d25618a764cf6e6564f6ef20eb288af3f9f52f
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 99cc1de..a19fc4c 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -925,7 +925,6 @@
 		hl_secure_hbw_rr_gen(rr_idx, min, max, 0, HL_SEC_PSOC_INST_NUM, HL_SEC_ENGINE_PSOC);
 }
 
-#define SPI_FLASH_SIZE		0x1000000ull /* 16MB */
 #define DBG_SPACE_BASE		0x1000007FF8000000ull
 #define SPI_FLASH_BASE		(STM_FLASH_BASE_ADDR + STM_FLASH_SIZE)
 #define SPI_FLASH_END		(SPI_FLASH_BASE + SPI_FLASH_SIZE)
